md/runtime-monitoring.md

```diff
@@ -0,0 +1,102 @@
+# Bonita Platform Monitoring
+
+Discover how to monitor a runtime environment running Bonita Platform 
+
+## Why monitoring ?
+
+Monitoring a Production environment is crucial to ensure the platform is correctly sized and tuned.
+
+Bonita provides a series of technical and BPM-related metrics to monitor the health of Bonita platform.  
+Some metrics are enabled by default and cannot be disabled. Some others are optional and can be enabled according to
+your needs.
+
+## Glossary
+
+**Work**: a unit piece of code that executes processes, tasks, BPM elements... and allows the processes to execute forwards.
+It executes in a Java thread.
+
+**Connector work**: a unit piece of code that specifically executes Bonita connectors
+
+**Work queue**: a queue storing pending works before they are taken by threads for execution 
+
+## Bonita-related metrics
+Bonita-related metrics are **enabled by default** and cannot be disabled. Here they are:
+* The number of currently running works, under the logical key name **org.bonitasoft.engine.work.works.running**
+* The number of currently pending works, waiting in the work queue to be treated, under the logical key name **org.bonitasoft.engine.work.works.pending**
+* The total number of executed works (since the last start of Bonita platform), under the logical key name **org.bonitasoft.engine.work.works.executed**
```

Comment:
Last time I tries, the name of the Query in graphana was `org_bonitasoft_engine_work_works_executed_total`, did you forget the total here or did you change it?

Reply:
I will check that.
:running_man:

Reply:
I don't know what's going on. I suppose Grafana exposes counter with another name, derived from the original one...

Reply:
yep. Multiple metrics are published for each `key` 
Since it depends on the monitoring system (micrometer registry) we can keep it like that